packages/expo-sms/android/src/main/java/expo/modules/sms/SMSModule.java

```diff
@@ -120,4 +110,16 @@ public void onHostPause() {
   public void onHostDestroy() {
     // do nothing
   }
+
+  private String constructSmsTo(List<String> addresses) {
+    if(addresses.size() > 0) {
+      final StringBuilder addressesBuilder = new StringBuilder(addresses.get(0));
+      for (int idx = 1; idx < addresses.size(); idx++) {
```

Comment:
`for (String address : addresses) {` might be better, we use this format in most places